### PR TITLE
Project 12 - Isabella Billgren

### PR DESCRIPTION
**Pull request by Bellsibub**
!!See ReadMe for project info!!

Also, instead of key moments (like my previous project) I made a Feature section for the ReadMe. I think that is more useful for reviewers and any developer looking over a project. 

But my reflections are included as always at the bottom of the ReadME.
 
Link to published project: https://filmoona.netlify.app/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,80 @@
...
+    * click on title => directs to `/`
+
+
+***
+## 💭 Reflections 💭
+All in all this project was really hard for me to start working on. I struggled alot with getting inspired to even begin. However half-way through the week I managed to pull myself together and get cracking!
+
+Once production had begun it was easy to get everything running. Also, I actually managed to create a custom hook for the fetch request function! 😃 So happy about this!
+<br><br>
+Issues that came up:
```
Comment (Sartish) [inline README.md:65]: Nice reflection points!

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+        </Route>
+        <Route path="/movies/:id">
+          <Movie />
+        </Route>
+        <Route path="/collections/:id">
+          <Collection />
+        </Route>
+        {/* If user tries to go to a route that is non-existent we redirect to main page */}
+        <Route>
+          <Redirect to="/" />
```
Comment (Sartish) [inline code/src/App.js:26]: Very good use of Route that you take usage of different information from the API. Super impressed🥇

File: code/src/components/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import RouteHeaderLink from 'components/Styled/RouteHeaderLink';
+
+const HeaderSection = styled.header`
+  background: var(--dark1);
+  display: flex;
+  justify-content: center;
+`;
```
Comment (Sartish) [inline code/src/components/Header/index.js:10]: nice way of styling🎨🙋‍♀️

File: code/src/components/LoaderSpinner/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        <Dot />
+        <Dot />
+        <Dot />
+        <Dot />
+        <Dot />
+        <Dot />
+      </Spinner>
+    </Wrapper>
+  );
+};
```
Comment (Sartish) [inline code/src/components/LoaderSpinner/index.js:18]: very nice spinner!

File: code/src/components/Rating/index.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const RatingCircle = styled.svg`
+  height: 100px;
+  width: 100px;
+  margin: 1em;
+  transform: rotate(270deg);
+  stroke-dasharray: 271; /* (2pie * radius) */
+  stroke-dashoffset: ${(props) => props.ratingVal};
```
Comment (Sartish) [inline code/src/components/Rating/index.js:10]: Such a cool way of showing the rating, So creative!!🤩

File: code/src/components/SectionDetails/index.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            ))}
+          </Pills>
+        )}
+        {other.vote_average ? (
+          <Rating rating={other.vote_average} />
+        ) : (
+          <i>No current voting average</i>
+        )}
+        <p>{overview}</p>
+      </DetailsText>
```
Comment (Sartish) [inline code/src/components/SectionDetails/index.js:35]: Good use of information from API, so cool to see this code, well done!

**Review by Sartish — APPROVED** [review]
Hi Bella!
Great application! As usual you have made a really impressive job, using both structured code and creative ideas. It is really inspiring to see. We didn't even find a wrong indentation, so will approve this project👏. And all the comments are relatid to how good we think it is ⭐️🙋‍♀️  So good use of styled components directly. very good job 🙌

**Review by KarolCislowski — APPROVED** [review]
I love that you are putting so much attention to learn style... You are one step from becoming CSS Demon 😈 Maybe it is time to jump to the SCSS? 🦄

**Pull request closed without merging**